### PR TITLE
Point extensions docs to .NET content

### LINKED ISSUE #20097: Add links to https://docs.microsoft.com/dotnet/core/extensions/
https://docs.microsoft.com/en-us/dotnet/core/extensions/configuration
For config, DI, GH, Logging, Options

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7184a308-a254-9350-a5cd-5bced1f369ae
* Version Independent ID: 726e3bf1-f367-d733-8933-bccc04da0e16
* Content: [Logging in .NET Core and ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-3.1)
* Content Source: [aspnetcore/fundamentals/logging/index.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/fundamentals/logging/index.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-fundamentals**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #20097

### CODE REVIEW

File: aspnetcore/fundamentals/host/generic-host.md [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
+description: Use .NET Core Generic Host in ASP.NET Core apps.  Generic Host is responsible for app startup and lifetime management.
 monikerRange: '>= aspnetcore-2.1'
 ms.author: riande
 ms.custom: mvc
 ms.date: 4/17/2020
 no-loc: ["ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
 uid: fundamentals/host/generic-host
 ---
-# .NET Generic Host
+# .NET Generic Host in ASP.NET Core
```
Comment (Rick-Anderson) [inline aspnetcore/fundamentals/host/generic-host.md:12]: HELP SEO. The .NET version right now is hard to find with goog search.

File: aspnetcore/fundamentals/host/generic-host.md [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
 
 ::: moniker range=">= aspnetcore-5.0"
 
 The ASP.NET Core templates create a .NET Core Generic Host (<xref:Microsoft.Extensions.Hosting.HostBuilder>).
 
+This topic provides information on using .NET Generic Host in ASP.NET Core. For information on using .NET Generic Host in a .NET app or .NET console app, see [.NET Generic Host](/dotnet/core/extensions/generic-host).
```
Comment (Rick-Anderson) [inline aspnetcore/fundamentals/host/generic-host.md:18]: @IEvangelist  I'm just getting started but would like to:

* Know the language you prefer.
* Drive traffic to your docs

Please suggest better wording I can use in all the extensions docs you've created. ASP.NET Core apps are .NET apps, so I'm not happy with the wording. I could go with something like 

for non-ASP.NET Core .NET apps
but that's worse.

Comment (IEvangelist) [reply aspnetcore/fundamentals/host/generic-host.md:18]: I prefer .NET apps, as they are all just .NET apps. I believe that @tdykstra had put together a doc that discussed which language to use when - Tom, could you please share that with Rick?

Comment (tdykstra) [reply aspnetcore/fundamentals/host/generic-host.md:18]: The naming document is about how to refer to .NET 5/Core in general, without regard to app type. My suggestion for this text is:

```suggestion
This topic provides information on using .NET Generic Host in ASP.NET Core. For information on using .NET Generic Host in console apps, see [.NET Generic Host](/dotnet/core/extensions/generic-host).
```

Also, I wouldn't limit this text to a .NET 5 moniker block, it should apply to 3.1 as well.

Comment (Rick-Anderson) [reply aspnetcore/fundamentals/host/generic-host.md:18]: > Also, I wouldn't limit this text to a .NET 5 moniker block, it should apply to 3.1 as well.

I'll add it to both once it's approved.

**Pull request merged**